### ISSUE TITLE
memory leak when closing player

### ISSUE THREAD

**Issue by brinoausrino** [bangnoise/ofxHapPlayer#57]
When closing the Player the DecodedFrame buffer remains with its allocated memory.

see https://github.com/bangnoise/ofxHapPlayer/pull/56 for a fix.

**Comment by bangnoise** [comment]
This isn't a leak. Regardless of what happens in std::vector::clear(), the memory is deallocated when the vector is destroyed when the DecodedFrame is destroyed when the Player is destroyed.

Are you running into an issue where you see otherwise?

**Comment by brinoausrino** [comment]
I would describe the case where I ran into that problem:
I created a Gallery with several texture thumbnails of hap videos. When clicking on a specific item the video is beeing loaded and played. When selecting another video I close that video and open another one. Each element has its own hapPlayer. When closing the video it clear the frame but it still reserves the 2MB of the video frame. So this adds up when having a look at many pictures and needs a lot of space. So I do not delete the player, only close the playback. I would would prefer not to delete the player instance until the image is not deleted from the gallery.
